kite-data/kite-data-core/src/main/java/org/kitesdk/data/View.java

```diff
@@ -109,6 +111,15 @@
   public Class<E> getType();
 
   /**
+   * Get the schema of entities contained in this {@code View}.
+   *
+   * @return the schema of entities contained in this view
+   *
+   * @since 1.1.0
+   */
+  public Schema getSchema();
```

Comment:
I think this should be named `getReadSchema` instead. The problem is that a dataset is also a view, so we need a way to distinguish between the dataset's schema and the schema you get when using it as a view. Otherwise it appears that I can change the dataset's schema using `asSchema`.